css/desktop.css

```diff
@@ -55,13 +55,18 @@ a {
 }
 
 .hero-section .main-nav-link-wrapper {
-  background-color: #d3d3d3;
+  background-color: white;
   display: grid;
-  grid-template-columns: 40% 60%;
+  grid-template-columns: 35% 60%;
+  gap: 5%;
   text-align: center;
   padding: 1% 10% 1% 10%;
 }
 
+.main-nav-link-wrapper .logo {
+  width: 100%;
+}
+
```

Comment:
- [ Optional ] I think it would be better if you made the `width` of the logo smaller( 200px), currently, it's too large which makes the  `menu` have unnecessary white spaces at the top and bottom sides. Please see the screenshot below, I highly recommend you do it for your design to be consistent with the design on Behance. 

## Your design
![Screenshot from 2023-05-25 02-22-20](https://github.com/Paulinelali/Microverse-capstone-one/assets/16392046/1a7abcd3-4685-436b-b7db-279821dd327a)

## Behance design
![Screenshot from 2023-05-25 02-28-30](https://github.com/Paulinelali/Microverse-capstone-one/assets/16392046/c2e51017-ce9e-4351-bc16-9b41e5c0edaf)